### PR TITLE
improve windows support

### PR DESCRIPTION
**Pull request by bbatha**
now builds on windows!

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -71,6 +71,8 @@ macro_rules! println_err(
     }}
 );
 
+const APP_INFO: app_dirs::AppInfo = app_dirs::AppInfo {name: "cargo-web", author: "Jan Bujack"};
```
Comment (koute) [inline src/main.rs]: You've misspelled my name. (:

File: src/main.rs [hunk 2]
```diff
@@ -238,8 +240,9 @@ fn download_package( package: &PrebuiltPackage ) -> PathBuf {
     let url = Url::parse( package.url ).unwrap();
     let package_filename = url.path_segments().unwrap().last().unwrap().to_owned();
 
-    let xdg_dirs = xdg::BaseDirectories::with_prefix( "cargo-web" ).unwrap();
-    let unpack_path = xdg_dirs.place_data_file( package.name ).unwrap().join( package.arch );
+    let unpack_path = app_dirs::get_app_dir(app_dirs::AppDataType::UserData, &APP_INFO, package.name )
```
Comment (koute) [inline src/main.rs]: 1. Missing space after the `(`. (:
2. According to `app_dirs` docs the `get_app_dir` doesn't create missing directories, so probably `app_dir` should be used instead?

File: src/main.rs [hunk 3]
```diff
@@ -377,6 +384,8 @@ fn check_for_emcc( use_system_emscripten: bool ) -> Option< PathBuf > {
         println_err!( "  sudo apt-get install emscripten" );
     } else if cfg!( target_os = "linux" ) {
         println_err!( "You can most likely find it in your distro's repositories." );
+    } else if cfg!( target_os = "windows" ) {
+        println_err!( "Download and install emscripted from the official site: http://kripken.github.io/emscripten-site/docs/getting_started/downloads.html")
```
Comment (koute) [inline src/main.rs]: Typo? `emscripted` -> `emscripten`

Also missing space before `)`. (: (I'll have to set up `rustfmt` sooner or later.)

File: src/main.rs (hunk 1 above)
Comment (bbatha) [inline src/main.rs]: doh!

**Comment by bbatha** [comment]
All outstanding issues fixed.

**Pull request merged**